### PR TITLE
data:AnikethDeb2004

### PR DESCRIPTION
**Pull request by AnikethDeb2004**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

### CODE REVIEW

File: data/AnikethDeb2004.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Aniketh Deb",
+  "displayStatsPublic": true,
```
Comment (sandramsc) [inline data/AnikethDeb2004.json:3]: This tag is no longer required.
```suggestion

```

File: data/AnikethDeb2004.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  "type":"personal",
```
Comment (sandramsc) [inline data/AnikethDeb2004.json:4]: This tag is no longer required.
```suggestion

```

File: data/AnikethDeb2004.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "bio": "I'm Aan open Source Contributor/Developer",
+  "avatar": "https://github.com/AnikethDeb2004.png",
```
Comment (sandramsc) [inline data/AnikethDeb2004.json:6]: This tag is no longer required.
```suggestion
```

**Review by sandramsc — CHANGES_REQUESTED** [review]
@AnikethDeb2004 I've left a few comments with some inline suggestions. Also perhaps consider adding some more links, milestones, testimonials to your profile in the future. 

Welcome to the community...


wlecome2biodrop

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**